deploy-eks/fb-editor-chart/templates/unpublish_test_services.yaml

```diff
@@ -0,0 +1,73 @@
+{{- if eq .Values.environmentName "test" }}
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: fb-editor-cron-unpublish-test-services-{{ .Values.environmentName }}
+  namespace: formbuilder-saas-{{ .Values.environmentName }}
+spec:
+  schedule: "0 20 * * FRI"
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 3
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          labels:
+            from-address: cronjob
+        spec:
+          containers:
+          - name: "fb-editor-workers-{{ .Values.environmentName }}"
+            image: "754256621582.dkr.ecr.eu-west-2.amazonaws.com/formbuilder/fb-editor-workers:{{ .Values.circleSha1 }}"
+            args:
+            - /bin/sh
+            - -c
+            - bundle exec rake unpublish:test_services
+            securityContext:
+              runAsUser: 1001
+            imagePullPolicy: Always
+            envFrom:
+              - configMapRef:
+                  name: fb-editor-config-map
+            env:
+              - name: SENTRY_DSN
+                valueFrom:
+                  secretKeyRef:
+                    name: fb-editor-secrets-{{ .Values.environmentName }}
+                    key: sentry_dsn
+              - name: SECRET_KEY_BASE
+                valueFrom:
+                  secretKeyRef:
+                    name: fb-editor-secrets-{{ .Values.environmentName }}
+                    key: secret_key_base
+              - name: DATABASE_URL
+                valueFrom:
+                  secretKeyRef:
+                    name: rds-instance-formbuilder-editor-{{ .Values.environmentName }}
+                    key: url
+              - name: EDITOR_SERVICE_ACCOUNT_TOKEN
+                valueFrom:
+                  secretKeyRef:
+                    name: {{ .Values.bearer_token }}
+                    key: token
+              - name: AWS_SES_ACCESS_KEY_ID
```

Comment:
Do we need the AWS secrets for this cronjob?

Reply:
good spot! i have removed them, and also taken them out of the test service configs rake task as well.